### ISSUE TITLE
libtimidity

### ISSUE THREAD

**Issue by BlyatOrange** [hilgenberg/amoc#1]
F
THIS
DEPENDENCY

jokes asside, I just spent hours trying to get it installed from source (only timidity++ is in the arch repos or aur) and kept getting hung up because of the lack of documentation, information, or from it simply not wanting to compile. However, I did find something that worked and wanted to share it hoping to help others with the same problem (at least until we're able to compile without specific audio libs). Unpacking the source code RPM provided in the Fedora repos, and compiling as one would normally, works perfectly. At least providing a link to the RPM and some brief information for the time being would be greatly apreciated. 
https://fedora.pkgs.org/34/fedora-x86_64/libtimidity-0.2.6-6.fc34.x86_64.rpm.html

**Comment by hilgenberg** [comment]
Right. That entire how-to-build thing is a bit shit, come to think of it ;-) I'll add your link and some details. But will probably take me until the weekend to get around to it.

**Comment by BlyatOrange** [comment]
That's all good. I've managed to get the library installed and compile amoc, but am still getting this:
`./amoc: error while loading shared libraries: libtimidity.so.2: cannot open shared object file: No such file or directory`
I've checked and it appears that libtimidity.so.2 has been properly installed in /usr/local/lib. could it be expected to be installed in some other location? Just to be sure; I'm running arch, x86, zen kernel, zsh, 17-7700K.

**Comment by hilgenberg** [comment]
Did you get it sorted? If not: Is /usr/local/lib/ really in your library path? If not: maybe look into /etc/ld.so.conf.d
